_includes/manuals/1.13/4.4.3_prov_templates.md

```diff
@@ -49,3 +51,36 @@ Finish templates are available for all hypervisors that support image based inst
 
 ###### User Data Templates
 User Data Templates are available for hypervisors that support customization via tools like *cloud-init*. In this case the installed machine does not need be reachable via ssh by the Foreman server. However, the installed must be able to reach Foreman or a Smart Proxy with the templates feature via http(s) to notify the setup has finished.
+
+###### PXE Loader
+
+When creating new Host, PXE Loader option must be selected in order to pass
+the correct DHCP filename option to the client. One option out of the
+following must be picked up:
+
+* PXELinux BIOS (loads `pxelinux.0` filename from TFTP)
+* PXELinux UEFI (loads `pxelinux.efi` filename from TFTP)
+* PXEGrub UEFI (loads `grub/grubx64.efi` filename from TFTP)
+* PXEGrub UEFI SecureBoot (loads `grub/shim.efi` filename from TFTP)
+* PXEGrub2 UEFI (loads `grub2/grubx64.efi` filename from TFTP)
+* PXEGrub2 UEFI SecureBoot (loads `grub2/shim.efi` filename from TFTP)
+* None - no filename passed (e.g. for HTTP booting via iPXE)
+
+Grub filenames are different for each individual architecture associated with
+the Host:
+
+* grubx64.efi or grubia32.efi (for Intel named "x86-64" or "ix86")
```

Comment:
I'd split this into two bullet points to match the rest, not sure they should be on the same line. Maybe use an asterisk instead of "x" in "ix86" - some of the "x" characters are literal, this is a wildcard.